x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/components/header_actions/syncs_context_menu.tsx

```diff
@@ -75,6 +77,13 @@ export const SyncsContextMenu: React.FC = () => {
   const shouldShowIncrementalSync =
     productFeatures.hasIncrementalSyncEnabled && hasIncrementalSyncFeature;
 
+  const isEnterpriseSearchNotAvailable =
```

Comment:
I think you need to wrap this one in Boolean (because errorConnectingMessage is a string), and not isSyncsDisabled on the next line.

Reply:
back in my day these things were easier !! :P